### ISSUE TITLE
missing extension in import paths in @babel/runtime/helpers/esm

### ISSUE THREAD

**Issue by jedwards1211** [babel/babel#12158]
## Bug Report

[File extensions are required in import statements in ES Modules.](https://nodejs.org/api/esm.html#esm_mandatory_file_extensions)

But files within `@babel/runtime/helpers/esm` (which are treated as ES Modules because `@babel/runtime/helpers/esm/package.json` contains `type: "module"`) don't use extensions in their import statements.

- [x] I would like to work on a fix!

**Current behavior**

I'm getting this correct error with Webpack 5:
```
ERROR in ./node_modules/@babel/runtime/helpers/esm/get.js 1:0-44
Module not found: Error: Can't resolve './superPropBase' in '/Users/andy/clarity/node_modules/@babel/runtime/helpers/esm'
Did you mean 'superPropBase.js'?
BREAKING CHANGE: The request './superPropBase' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```
This is because the first line of `@babel/runtime/helpers/esm/get.js` is an import without an extension:
```
import superPropBase from "./superPropBase";
```
**Input Code**

```js
var your => (code) => here;
```

**Expected behavior**

No problems using `@babel/plugin-transform-runtime` with `useESModules: true`

**Babel Configuration**
- Filename: (babel config inlined in babel-loader webpack config)

```js
{
  presets: [
    ['@babel/preset-env', {targets: {browsers: 'last 2 versions'}}],
  ],
  plugins: [
    [
      '@babel/plugin-transform-runtime',
      {
        useESModules: true,
      },
    ],
  ]
}
```

**Environment**

```

```
- `@babel/runtime`: 7.11.2
- `@babel/plugin-transform-runtime`: 7.11.5
- `webpack`: 5.0.0-rc.4
- `babel-loader`: 8.1.0

**Possible Solution**

Use full extensions in imports inside `@babel/runtime/helpers/esm`, so I don't get these errors, for example:
```
import superPropBase from "./superPropBase.js";
```

**Comment by jedwards1211** [comment]
Seems like this will be fixed by #10853

**Comment by JLHwung** [comment]
Duplicate of #12003

**Comment by JLHwung** [comment]
Duplicate of #8462

**Comment by JLHwung** [comment]
Fixed in `@babel/runtime@7.12.0`.
